### PR TITLE
MCPClient: fix dip_generation_helper.py logging

### PR DESCRIPTION
**Pull request by hakamine**

### CODE REVIEW

**Comment by hakamine** [comment]
Found errors. Will fix and re-push

File: src/MCPClient/lib/clientScripts/dip_generation_helper.py [hunk 1]
```diff
@@ -36,9 +39,11 @@ def create_archivesspace_client():
             repository=config['%repository%']
         )
     except archivesspace.AuthenticationError:
+        logging.error('Unable to authenticate to ArchivesSpace server using the default user! Check administrative settings.')
```
Comment (Hwesta) [inline src/MCPClient/lib/clientScripts/dip_generation_helper.py]: These should be `logger` not `logging`.  `logging` logs to the root logger, while `logger` logs to the define logger (below), and it's very useful for determining where a log originated and getting it filtered to the correct log files to use defined loggers.

**Review by sevein — APPROVED** [review]
@hakamine I've fixed the issue where the logger was defined but the you were still using the default one. I think this is ready to merge but please take a look!

**Comment by hakamine** [comment]
Thank you for fixing! Looks good to me :+1:

**Pull request merged**